cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -160,8 +175,7 @@ class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
     val p = Map.empty[String, String]
     snc.createTable(tableName, "column", dataDF.schema, p)
 
-    val tName = s"${Constant.INTERNAL_SCHEMA_NAME}.${
-      tableName.toUpperCase}${Constant.SHADOW_TABLE_SUFFIX}"
+    val tName = ColumnFormatRelation.cachedBatchTableName(tableName.toUpperCase())
```

Comment:
ok. will update this

On Sun, May 22, 2016 at 2:41 AM, Sumedh Wale <notifications@github.com>
wrote:

> In
> cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala
> <https://github.com/SnappyDataInc/snappydata/pull/251#discussion_r64140315>
> :
>
> > @@ -160,8 +175,7 @@ class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
> >      val p = Map.empty[String, String]
> >      snc.createTable(tableName, "column", dataDF.schema, p)
> >
> > -    val tName = s"${Constant.INTERNAL_SCHEMA_NAME}.${
> > -      tableName.toUpperCase}${Constant.SHADOW_TABLE_SUFFIX}"
> > +    val tName = ColumnFormatRelation.cachedBatchTableName(tableName.toUpperCase())
>
> @nthanvi <https://github.com/nthanvi> Missed this in initial review. Both
> original and updated code should never be using toUpperCase which breaks
> case-sensitivity SQLConf setting. Can you check precheckin removing
> toUpperCase? I believe all case-sensitivity issues should be now fixed in
> current code.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SnappyDataInc/snappydata/pull/251/files/e47bf44b93f99520a650851f5de7676e56c5188e#r64140315>
>